program/Core/Server/functions.sh

```diff
@@ -76,7 +76,7 @@ Core.Server::requestStart () {
 
 	# LAUNCH! (in tmux)
 
-	tmux -f "$THIS_DIR/cfg/tmux.conf" -S "$SOCKET" new-session -n "server-control" -s "$APP@$INSTANCE" /bin/bash "$INSTANCE_DIR/msm.d/tmp/server-control.sh" \; detach
+	tmux -f "$THIS_DIR/cfg/tmux.conf" -S "$SOCKET" new-session -n "server-control" -s "$APP@$INSTANCE" -d /bin/bash "$INSTANCE_DIR/msm.d/tmp/server-control.sh" \; detach
```

Comment:
I don't exactly understand how this is necessary for systemd support. However, the '-d' flag is still useful in that place.

With this change, the last part ` \; detach` of the command should not be necessary anymore. Can you remove that part and check if it still works?
Thanks!

Reply:
The daemon flag is required because when starting the server via a [SystemD](https://systemd.io) service without it the following error is encountered:
```
open terminal failed: not a terminal
```

Also it still works with the detach command removed.